Makefile

```diff
@@ -18,6 +17,10 @@ dependencies:
 	go get -u github.com/spf13/cobra
 	go get -u github.com/stretchr/graceful
 	go get -u golang.org/x/crypto/twofish
+dev-dependencies: dependencies
+	go install -race std
+	go get -u github.com/dvyukov/go-fuzz/go-fuzz
+	go get -u github.com/dvyukov/go-fuzz/go-fuzz-build
```

Comment:
github didn't detect this as an outdated diff, but if you check, you'll see that I fixed the issue.